### PR TITLE
feat(Header): Prevent page from scrolling when header is open

### PR DESCRIPTION
**Pull request by kosanna**

### CODE REVIEW

**Comment by kosanna** [comment]
This one is done

File: react/private/smallDeviceOnly.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import get from 'lodash/get';
+import invoke from 'lodash/invoke';
+
+const responsiveBreakpoint = 740;
+const smallDeviceOnlyMedia = `(max-width: ${responsiveBreakpoint - 1}px)`;
+const getMatchMedia = typeof window !== 'undefined' && invoke(window, 'matchMedia', smallDeviceOnlyMedia);
+
+export default () => get(getMatchMedia, 'matches');
```
Comment (tamoore) [inline react/private/smallDeviceOnly.js]: This is pretty cool!

**Comment by markdalgleish** [comment]
What happens if the menu is taller than the viewport?

**Comment by kosanna** [comment]
Good point, I'll need to add isolated scroll to it

**Pull request merged**